deltachat-ios.xcodeproj/project.pbxproj

```diff
@@ -949,7 +963,12 @@
 				305FE03523A81B4C0053BE90 /* EmptyStateLabel.swift */,
 				AEFBE22E23FEF23D0045327A /* ProviderInfoCell.swift */,
 				AEB54C7E246DBA610004624C /* FlexLabel.swift */,
+<<<<<<< HEAD
```

Comment:
This needs to be deleted, remains of a merge conflict

Reply:
> There's a mismatch between the image ordering of the grid view controller and the detail image controller

Yeah true. I reverted the messageIds ordering (show newest on top). This might be the reason. Thank you!